### ISSUE TITLE
Undefined test case name

### ISSUE THREAD

**Issue by fadc80** [fadc80/karma-sonarqube-reporter#10]
After commit d30a5250c3b939949afad35d6d44f2c2439298ac test case names are defined using the property `result.fullName` provided by [karma-jasmine](https://github.com/karma-runner/karma-jasmine). However, this property was introduced only on version 1.1.2 (latest). If we use a previous one, then test case names are reported as undefined.

```xml
<?xml version='1.0'?>
<testExecutions version='1'>
    <file path='src/app/app.component.spec.ts'>
        <testCase name='undefined' duration='1'/>
    </file>
</testExecutions>
```

**Comment by cslecours** [comment]
The fix is not present in the latest npm version (1.2).

**Comment by fadc80** [comment]
Yeah. I've just published version 1.2.1. Check it out.
